config.py

```diff
@@ -8,8 +8,8 @@
 # Application settings
 
 # API metadata
-API_TITLE = 'Model Asset Exchange Server'
-API_DESC = 'An API for serving models'
+API_TITLE = 'Object Detector'
```

Comment:
I think we should make this MAX Object Detector, that's how we typically refer to it in other documentation

Reply:
Sure, once we have confirmation for your other comment I will make the changes. Thanks!